modules/internal/ChapelRange.chpl

```diff
@@ -3052,17 +3252,27 @@ operator :(r: range(?), type t: range(?)) {
     return isEnumType(t) && t.size == 1;
   }
 
-  proc chpl__defaultLowBound(type t) {
+  proc chpl__defaultLowBound(type t, boundedType: BoundedRangeType) {
```

Comment:
I ended up making it `param` rather than `inline`, which seems even better.  I also made it `private`.  Making it a single routine seems orthogonal to this effort given that it was pre-existing, and doing so would mean it couldn't be `param` anymore, which I think is a net win.

I will add a comment if you want—I didn't initially since it was pre-existing (and by the time I finished the PR, I couldn't remember what it did without looking at the callsites anyway).

Reply:
I like param. A comment will be nice, however not a must. We can write the comment next time when it matters.